setup.py

```diff
@@ -10,8 +10,8 @@
 from setuptools import Extension, setup
 
 
-if sys.version_info < (3, 5, 3):
-    raise RuntimeError("aiohttp 3.x requires Python 3.5.3+")
+if sys.version_info < (3, 6):
```

Comment:
It's okay to just remove this check since setuptools that supports `python_requires` is shipped with Python 3.6.

Reply:
I want to fail loudly even on Python 3.5